src/App.tsx

```diff
@@ -26,70 +27,133 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
   // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  const visibleGoods: string[] = [...goods];
+
+  visibleGoods.sort((firstGood, secondGood) => {
+    switch (sortType) {
+      case SortType.LENGTH:
+        return firstGood.length - secondGood.length;
+
```

Comment:
Don't start sorting if you don't have sort type

 switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )

Reply:
Fixed.